paasta_tools/cli/cmds/logs.py

```diff
@@ -81,7 +84,7 @@ def add_subparser(subparsers):
         help=cluster_help,
     ).completer = completer_clusters
     status_parser.add_argument(
-        '-f', '-F', '--tail', dest='tail', action='store_true', default=True,
+        '-f', '-F', '--tail', dest='tail', action='store_true', default=False,
```

Comment:
Do we really want to couple this default change to the addition of support for non-tailing? I would think this would make a better follow-up change after we let people test out this new functionality a bit (and update all of our documentation).